go/tosca/interpreter_registry.go

```diff
@@ -30,14 +30,47 @@ import (
 // GetInterpreter performs a lookup for the given name (case-insensitive) in
 // the registry. The result is nil if no interpreter was registered under the
 // given name.
+//
+// Deprecated: Use NewInterpreter instead.
```

Comment:
I could not find any use, can we remove it?

Reply:
Aida is using it.